tests/PySys/c8y_restart_bridge/pysystest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+
+  <description>
+	<title></title>
```

Comment:
As always, at least adding the title would be good.

Reply:
Right, somehow missed to add them.